### DISCUSSION TITLE
Generate npm provenance statement for releases

### DISCUSSION THREAD

**Discussion by aeharding** [radashi-org/radashi#292] [Ideas]
Hi there!

I would like to see radashi support [npm provinence](https://docs.npmjs.com/generating-provenance-statements) for npm releases. :)

**Comment by aleclarson** [comment]
I've added `--provenance` to our publishing script: a9f9a60

That script will run in GitHub CI.

Now I'm curious if we can disable `npm publish` without `--provenance` 🤔

**Reply by aeharding** [reply]
Awesome!!

Possibly by adding `provenance=true` to `.npmrc`?

**Reply by aleclarson** [reply]
@aeharding Perfect, thanks!

**Comment by aleclarson** [comment]
Seems to be working 😄 

<img width="229" alt="Screenshot 2024-11-09 at 15 44 47" src="https://github.com/user-attachments/assets/cd57b89b-3e24-42e6-bc13-5db3a5898c80">
